libraries/DataFlash/LogFile.cpp

```diff
@@ -1668,42 +1673,42 @@ bool DataFlash_Backend::Log_Write_Mode(uint8_t mode, uint8_t reason)
 // Write ESC status messages
 void DataFlash_Class::Log_Write_ESC(void)
```

Comment:
We should make this a generic method, not dependent on UAVCAN - I'm sure that soon we'll have other ESCs giving back that info into ArduPilot. Either we do a generic AP_ESC library or we include this info into SRV_Channel.

Reply:
Is it OK for now that the Log_Write_ESC method will have full parameters and will called from UAVCAN for now, but obviously can be called from other sources when they appear?